<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Parvathy" failures="1" tests="1" name="Test" time="35.487" errors="0" timestamp="2025-04-20T11:26:47 IST">
  <testcase classname="testCases.TC002_Login" name="Login_test" time="15.213">
    <failure type="java.lang.AssertionError" message="Login not successful expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login not successful expected [true] but found [false]
at testCases.TC002_Login.Login_test(TC002_Login.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Login_test -->
</testsuite> <!-- Test -->
